kolibri/auth/models.py

```diff
@@ -1,22 +1,34 @@
 """
```

Comment:
This file may be easiest to look at as a whole (the new version) rather than a diff, as per preference.